nixos/modules/services/misc/persistent-evdev.nix

```diff
@@ -0,0 +1,64 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.persistent-evdev;
+  settingsFormat = pkgs.formats.json {};
+
+  configFile = settingsFormat.generate "persistent-evdev-config" {
+    cache = "/var/cache/persistent-evdev";
+    devices = mapAttrs (virt: phys: "/dev/input/by-id/${phys}") cfg.devices;
+  };
+in
+{
+  options.services.persistent-evdev = {
+    enable = mkEnableOption "virtual input devices that persist even if the backing device is hotplugged";
+
+    devices = mkOption {
+      default = {};
+      type = types.attrsOf types.str;
+      description = ''
+        A set of virtual proxy device labels with backing physical device ids.
+        </para>
+        <para>
```

Comment:
You don't need this because this is done in [documentation postprocessing](https://github.com/NixOS/nixpkgs/blob/4042f7b3752e1420b40eb6e337d187cea666c1e3/nixos/lib/make-options-doc/postprocess-option-descriptions.xsl#L22-L24) already.